third_party/tcplp/Makefile.am

```
@@ -29,6 +29,7 @@
 include $(abs_top_nlbuild_autotools_dir)/automake/pre.am
 
 EXTRA_DIST                                    = \
+    lib                                         \
     tcplp.h                                     \
```

I think you need to do the following to get `distcheck` to pass.
```suggestion
    tcplp.h                                     \
    lib/bitmap.h                                \
    lib/cbuf.h                                  \
    lib/lbuf.h                                  \
```

---

Thanks! I've included this in my latest update.